### PR TITLE
net/bluetooth: Test bluetooth enable feature using bt_enable

### PR DESCRIPTION
**Pull request by amitpurwar**
Signed-off-by: amit purwar <amit.purwar@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [43040904a411bf2132bb8625b85849f473f7356a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [43040904a411bf2132bb8625b85849f473f7356a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [156e08f1d87705bf40cddbf259e637e01dbeea5f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [156e08f1d87705bf40cddbf259e637e01dbeea5f] - Code Rule Check OK.

File: apps/examples/bluetooth/bt_unit_test.c [hunk 1]
```diff
@@ -23,8 +23,24 @@
...
+void bt_enable_cb(int err)
+{
+	printf("bt_enable() callback with error code:[%d] \n", err);
+}
+
 int bt_unit_test_main(int argc, char **argv)
 {
+	int ret = -1;
+
 	printf("Enter bt_unit_test_main()\n");
```
Comment (Wootak) [inline apps/examples/bluetooth/bt_unit_test.c]: I suggest to use PRT() in bt_unit_test.h.
PRT("Enter bt_unit_test_main()\n");

File: os/net/bluetooth/bluetooth.c [hunk 2]
```diff
@@ -71,7 +71,7 @@ struct bt_ad {
 
 /* Work structures: One for high priority and one for low priority work */
 
-static struct work_s g_hip_work;
+static struct work_s g_lowp_work;
```
Comment (Wootak) [inline os/net/bluetooth/bluetooth.c]: Could you tell me why you change this variable to low?

Comment (amitpurwar) [reply os/net/bluetooth/bluetooth.c]: bluetooth subsystem was not getting the response of HCI request from btnull driver as hci_rx_work was pending and init_work was running with hight priority

**Comment by seinfra** [comment]
Target : [1b249ccf0bee89aaf9ff83282de75f0c776374c9] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [1b249ccf0bee89aaf9ff83282de75f0c776374c9] - Code Rule Check (C++) OK.

**Pull request merged**